Docker.md

```diff
@@ -0,0 +1,26 @@
+## development docker-compose.yaml file
+
+### Publishing ports
+
+By default, containers within the same docker-compose network are able to connect to services running
+on other containers on any port by using the container name, as long as the container is listening
```

Comment:
```suggestion
on other containers on any port by using the container name, as long as the service is listening
```

Reply:
Proposed to replace with "the service running in the container" in https://github.com/metabrainz/guidelines/pull/18/commits/f087b462cae59ecf958cf9d03aaacf6c0dce2018 to distinguish from the [service configuration](https://docs.docker.com/compose/compose-file/compose-file-v3/#service-configuration-reference) in a Docker Compose project.